yospace/README.md

```diff
@@ -89,3 +89,10 @@ await yospaceConnector.setupYospaceSession(source, sessionProperties);
 ```
 
 Once the setup of the Yospace session is done, you can continue to use the player and the connector will handle everything related to Yospace.
+
+To shutdown the current Yospace session, use the connector's `destroy()` method. The connector automatically calls `session.shutdown()` on session initialization errors and when creating a new session. If you, however, set a non-yospace source to the player after using this connector, it is your responsibility to do this through `destroy()`.
```

Comment:
Do we really need a new connector method for this? Can't the connector call `reset()` automatically, for example when THEOplayer fires a `sourcechange` or `destroy` event? 🤔